test/msw-api/cli/init.test.ts

```diff
@@ -44,18 +45,45 @@ test('copies the worker script into an existing directory without errors', async
   await cleanup()
 })
 
-test('throws an exception and does not copy the worker script given a non-existing directory', async () => {
+test('creates the directory if it does not exist', async () => {
+  const { prepare, getPath, cleanup } = createTeardown(
+    'tmp/cli/init/non-existing',
+  )
+  await prepare()
+
+  const { stderr } = await promisifyChildProcess(
+    exec(`node cli/index.js init ${getPath('non-existing-public')} --no-save`),
```

Comment:
I suggest we provide a nested non-existing directory path so we also assert that nested directories are created.
```suggestion
    exec(`node cli/index.js init ${getPath('public/nested/path')} --no-save`),
```

Reply:
Changed to `public/nested/path`